modules/phase_field/src/auxkernels/NonlinPrecondAux.C

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NonlinPrecondAux.h"
+
+registerMooseObject("MooseApp", NonlinPrecondAux);
```

Comment:
this should be phase field app not MooseApp

this is why SQA is failing

Reply:
(eg because it sees mooseApp, it looks in framework/doc instead of modules...doc